sources/osg/CullVisitor.js

```diff
@@ -452,15 +453,22 @@ define( [
     // since the only difference is that we want to push an additional state
     // Maybe it will be useful when we'll add morph target geometry or something...
     var postPushGeometry = function ( cull, node ) {
-        if ( node.typeID === RigGeometry.typeID ) {
+        if ( node.typeID === RigGeometry.typeID || node.typeID === MorphGeometry.typeID ) {
```

Comment:
we will probably use instanceof , it will simplify the code and after some talk with stephane it's fast